src/server/node_types.rs

```diff
@@ -106,3 +110,14 @@ pub struct VariableNode {
     pub type_definition: ua::NodeId,
     pub attributes: ua::VariableAttributes,
 }
+
+pub struct MethodNode {
```

Comment:
@sgoll  What do you think how a MethodNode struct should be handled? 
It would be possible to use `Node` struct and a method node specific struct or just a struct that has all the required fields, like you can see here.